SmartCar/Arduino/sketch_main/sketch_main.ino

```diff
@@ -117,6 +144,9 @@ void printTemperature(DeviceAddress deviceAddress)
 void loop() { 
     //Is true when a command with a ? in the end is given or if the input reaches a length atleast 4
     if(completeCommand){
+
+      // Improves readability.
+      int value = input.substring(1, 4).toInt();
```

Comment:
How about just calling it input.substring(1, 4).toInt(); since that was what it was before. ANd that was clear to understand